.github/workflows/docker_build.yml

```diff
@@ -0,0 +1,90 @@
+name: Docker Build
+on:
+  push:
+    branches:
+      - master
+      - 'build/**'
+
+jobs:
+  draft-release:
+    runs-on: ubuntu-latest
+    outputs:
+      tag_name: ${{ steps.release_drafter.outputs.tag_name }}
+    steps:
+      - uses: release-drafter/release-drafter@v5
+        id: release_drafter
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+  docker:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+    needs: draft-release
+    env:
+      REGISTRY: ghcr.io
+      IMAGE_NAME: ${{ github.repository }}
+    strategy:
+      fail-fast: false
+      matrix:
+        platform:
+          - linux/amd64
+          - linux/amd64/v2
+          - linux/amd64/v3
+          # TODO: Build for other platforms. Will likely need our own
+          # runners for each of these since QEMU segfaults for most of
```

Comment:
Weird, do you know why it segsfaults?

Reply:
Yeah it happens due to the OTP 25 JIT.

Hmm. I can build it with the jit, and we can probably have cross platform then. Let me give that a shot.

Reply:
https://gitlab.com/qemu-project/qemu/-/issues/1034

Reply:
Building a new base image here:

https://github.com/firezone/containers/actions/runs/3131631443

Reply:
Seems like it failed due to broken github actions

Reply:
But with latest main it did work, so yeah.

I wonder if there's a significant loss of performance by disabling JIT?